### PR TITLE
Terms conds

### PR DESCRIPTION
**Pull request by PolyProgrammist**

### CODE REVIEW

File: src/lib.rs
Comment (frol) [inline src/lib.rs]:
```suggestion
            "Terms and conditions version cannot be from the future"
```

File: tests/proposals.rs
Comment (frol) [inline tests/proposals.rs]: Since we don't have printlns anywhere else, let's clean this one up

```suggestion
```

**Review by frol — DISMISSED** [review]

**Review by frol — DISMISSED** [review]

**Comment by frol** [comment]
@PolyProgrammist Consider playing with @race-of-sloths

**Comment by race-of-sloths** [comment]
@PolyProgrammist Thank you for your contribution! Your pull request is now a part of the [Race of Sloths](https://race-of-sloths.com)!
New Sloth joined the Race! Welcome!

<picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://badge.race-of-sloths.com/PolyProgrammist?type=bot&theme=dark">
    <source media="(prefers-color-scheme: light)" srcset="https://badge.race-of-sloths.com/PolyProgrammist?type=bot&theme=light">
    Shows an illustrated sun in light mode and a moon with stars in dark mode.
</picture>




<details><summary>Current status: <i>waiting for finalization</i></summary>

The pull request is merged, you have 24 hours to finalize your scoring. The scoring ends Thu Jul 11 15:10:57 2024
| Reviewer | Score |
|--------|--------|
| @frol  | 3 |

**The average score is 3**

</details><details><summary>What is the Race of Sloths</summary>
<p>
Race of Sloths is a friendly competition where you can participate in challenges and compete with other open-source contributors within your normal workflow

For contributors:

- Tag @race-of-sloths inside your pull requests
- Wait for the maintainer to review and score your pull request
- Check out your position in the [Leaderboard](https://race-of-sloths.com/leaderboard)
- Keep weekly and monthly streaks to reach higher positions
- Boast your contributions with a dynamic picture of your [Profile](https://race-of-sloths.com/profile/PolyProgrammist)

For maintainers:
- Score pull requests that participate in the Race of Sloths
- Engage contributors with fair scoring and fast responses so they keep their streaks
- Promote the Race to the point where the Race starts promoting you
- Grow the community of your contributors

Feel free to check [our website](https://race-of-sloths.com) for additional details!
</p>
</details> 

<details>
<summary>Bot commands</summary>

- For contributors
  - **Include a PR:** `@race-of-sloths include` to enter the Race with your PR
- For maintainers:
  - **Assign points:** `@race-of-sloths score [1/2/3/5/8/13]` to award points based on your assessment.
  - **Reject this PR:** `@race-of-sloths exclude` to send this PR back to the drawing board.
  - **Exclude repo:** `@race-of-sloths pause` to stop bot activity in this repo until `@race-of-sloths unpause` command is called

</details>

**Comment by frol** [comment]
@race-of-sloths 3

**Comment by race-of-sloths** [comment]
### ❓Unknown command 
Please check the command and try again

**Comment by frol** [comment]
@race-of-sloths score 3

**Comment by race-of-sloths** [comment]
### 🌟 Score recorded!

@frol, thank you for scoring this pull request in the Race of Sloths!

**Review by frol — DISMISSED** [review]

**Comment by race-of-sloths** [comment]
### ✅ PR is finalized! 
Your contribution is much appreciated with a final score of 3!
You have received 40 (30 base + 10 weekly bonus) Sloth points for this contribution


Another weekly streak completed, well done @PolyProgrammist! To keep your weekly streak and get another bonus make pull request next week! Looking forward to see you in race-of-sloths

**Pull request merged**
